Resolve-PendingMigrations.ps1

```diff
@@ -29,9 +29,9 @@ foreach ($mailbox in $allMailboxes) {
             Set-RemoteMailbox $mailboxInfo.UserPrincipalName -EmailAddresses $mailboxInfo.EmailAddresses `
              -EmailAddressPolicyEnabled $false
         } else {
-            Write-Host "Mailbox" $mailbox "has on-premise archive"
+            Write-Warning "Mailbox $mailbox has on-premise archive"
         }
     } else {
-        Write-Host "Mailbox" $mailbox "has on-premise content"
+        Write-Warning "Mailbox $mailbox has on-premise content"

```

Comment:
You could enclose the variable in single quotes, so the output is clearer for the user.

Reply:
Do you feel it helps?

PS C:\windows\system32> Write-warning -Message "mailbox $mailbox has onprem archive"
WARNING: mailbox user@domain.com has onprem archive
PS C:\windows\system32> Write-warning -Message "mailbox '$mailbox' has onprem archive"
WARNING: mailbox 'user@domain.com' has onprem archive

Reply:
Oh, didn't know that that's an email address.

Reply:
It isn't. It's usually just the alias like
`WARNING: mailbox 'vo456457' has onprem archive`

Reply:
Too late 😂 It's ok though; at least it doesn't contain spaces.

Reply:
![image](https://user-images.githubusercontent.com/39416712/103463137-d0b29480-4d32-11eb-8bd4-fa4d3d353f6e.png)

Reply:
![image](https://user-images.githubusercontent.com/39416712/103463156-fb045200-4d32-11eb-9115-d270591f2fb5.png)